riscv-executor/src/lib.rs

```diff
@@ -47,13 +47,15 @@ impl Elem {
     }
 
     // Rust doesn't allow us to implement From<F> for Elem...
+    // Panics if the value can't be represented as i64.
```

Comment:
I think this needs more explanation about how the field element is represented as a i64